app/views/research_sessions/preview.html.erb

```diff
@@ -76,13 +76,13 @@
   <section>
     <h3 class="subtitle-small" id="doi">Do I have to take part?</h3>
     <p>
-      Taking part is entirely voluntary - it is up to you to decide whether or not
-      to take part. If you decide to take part you do not have to answer questions
-      you do not want to answer. You can also change your mind about taking part
+      Taking part is entirely voluntary - it is up to you to decide whether or not <%= @research_session.able_to_consent? ? 'you' : 'your child/the child in your care' %>
```

Comment:
I'd be inclined to DRY these up by putting a method in `ResearchSessionsHelper`. Rails is opinionated on getting this stuff out of the view, so you could either add it to the `ResearchSessionsPresenter`, in which case it might look like `<%= @research_session.you_or_your_child %>` but it's terser and more readable (despite the lack of OO purity!) to just add it to `ResearchSessionsHelper` and reference `@research_session` therein:

`<%= you_or_your_child %>`

and in `ResearchSessionsHelper`:

```
  def you_or_your_child
    @research_session.able_to_consent? ? 'you' : 'your child/the child in your care'
  end
```